docs/curating/features.rst

```diff
@@ -0,0 +1,72 @@
+Curating Features
+=================
+Features are a top level entity in CIViC, enabling the classification of Variants into different classes.
+
+.. thumbnail:: /images/figures/Features1.png
+   :alt: Screenshot of an Evidence Item
+   :title: Figure 1: Screenshot of an Evidence Item
```

Comment:
This title will be displayed underneath the figure. Both this and the next figure have the same title and obth labeled as "Figure 1". It looks like you have built the caption (Figure legend) embedded in some of these photos.

Reply:
See source page: https://civic.readthedocs.io/en/latest/_sources/curating/genes.rst.txt for this page https://civic.readthedocs.io/en/latest/curating/genes.html The figure caption comes from the command below (not embedded in the image)

:title: The CIViC gene page shows a curator Description and a list of Molecular Profiles containing variants assoicated to the gene

Reply:
It looks like this problem persists throughout the new docs. For example Figures 3 and 4 also have the same Figure 1 title. Figures 5 and 6 share the same Figure 2 title, etc. The titles don't seem to match the figure contents.